build/rocm/Dockerfile.rocm

```diff
@@ -85,7 +85,6 @@ RUN add-apt-repository ppa:deadsnakes/ppa && \
 RUN update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.6 1 && \
      update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.9 2
 
-RUN pip3 install --upgrade setuptools pip
-  
-RUN pip3 install absl-py numpy==1.19.5 scipy wheel six setuptools pytest pytest-rerunfailures
+RUN pip3 install --upgrade --force-reinstall setuptools pip
   
+RUN pip3 install absl-py numpy==1.19.5 scipy wheel six setuptools pytest pytest-rerunfailures matplotlib
```

Comment:
FYI: We've dropped support for NumPy 1.19 in JAX. Update to 1.20?

Reply:
Done.  Thanks for the suggestion.